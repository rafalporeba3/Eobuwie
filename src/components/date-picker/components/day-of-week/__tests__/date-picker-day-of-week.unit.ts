import { shallowMount, Wrapper } from '@vue/test-utils';
import cDatePickerDayOfWeek      from '../date-picker-day-of-week.component.vue';

describe('Date picker day of week component', () => {
  let wrapper!: Wrapper<any>;

  beforeEach(() => {
    wrapper = shallowMount(cDatePickerDayOfWeek, {
      propsData: {
        dayOfWeek: 'Mon',
      },
    });
  });

  it('should render element with correct class', () => {
    expect(wrapper.classes())
      .toContain('c-date-picker-day-of-week');
  });

  it('should match snapshot', () => {
    expect(wrapper.html())
      .toMatchSnapshot();
  });
});
