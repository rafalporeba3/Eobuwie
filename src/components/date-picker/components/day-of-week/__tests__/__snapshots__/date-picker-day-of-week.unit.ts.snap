// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date picker day of week component should match snapshot 1`] = `<span class="c-date-picker-day-of-week">Mon</span>`;
